dandi/tests/fixtures.py

```diff
@@ -312,3 +317,63 @@ def upload_dandiset(paths=None, **kwargs):
         "dandiset_id": dandiset_id,
         "reupload": upload_dandiset,
     }
+
+
+@pytest.fixture(scope="session")
+def download_video_files(tmp_path_factory):
+    from datalad.api import install
+
+    pt = tmp_path_factory.mktemp("ophys_testing_data")
+    dataset = install(
+        path=pt, source="https://gin.g-node.org/CatalystNeuro/ophys_testing_data"
+    )
+    resp = dataset.get("video_files")
+    return Path(resp[0]["path"])
+
+
+@pytest.fixture()
+def create_video_nwbfiles(download_video_files, tmp_path):
+    base_nwb_path = tmp_path / "nwbfiles"
+    base_nwb_path.mkdir(parents=True, exist_ok=True)
+    base_vid_path = tmp_path / "video_files"
+    base_vid_path.mkdir(parents=True, exist_ok=True)
+    shutil.copytree(download_video_files, str(base_vid_path))
```

Comment:
`shutil.copytree()` can't copy to a directory that already exists.  Remove the `base_vid_path.mkdir(...)` line.